samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitoring.StartupProjects.txt

```diff
@@ -1,2 +1,3 @@
 EndpointsMonitor\EndpointsMonitor.csproj
-NServiceBusEndpoint\NServiceBusEndpoint.csproj
\ No newline at end of file
+NServiceBusEndpoint\NServiceBusEndpoint.csproj
+PlatformLauncher\PlatformLauncher.csproj
```

Comment:
In Monitoring, I called it Platform. Should we align on the name and also switch monitoring to  `PlatformLauncher`?

Reply:
I really don't think it matters.